code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -13,7 +13,7 @@
 /obj/item/mecha_parts/mecha_equipment/drill/action(atom/target)
 	if(!action_checks(target))
 		return
-	if(istype(target, /turf) && !istype(target, /turf))
```

Comment:
blame whoever wrote `if(istype(target, /turf) && !istype(target, /turf/simulated)) ` in the first place